R/bundle.R

```diff
@@ -80,7 +82,24 @@ bundleApp <- function(appName, appDir, appFiles, appPrimaryDoc, assetTypeName,
   bundlePath
 }
 
-isShinyRmd <- function(filename) {
+appHasParameters <- function(appDir, files) {
+  rmdFiles <- grep("^[^/\\\\]+\\.rmd$", files, ignore.case = TRUE, perl = TRUE,
```

Comment:
Are parameters supported for R Markdown documents other than the primary one in an application?